ecosystem/sep-0038.md

```diff
@@ -0,0 +1,176 @@
+
+`currencies` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`currency` | string | The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) code for the currency.
+`delivery_methods` | array | An array of objects describing the methods a client can use to deliver funds to the anchor for a given `currency`. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response. The array can be empty if the method of delivery is irrelevant to the anchor.
+
+`delivery_methods` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`name` | string | The value to use when making `POST /quote` requests.
+`description` | string | A human readable description of the method identified by `name`.
+
+### GET Prices
+
+This endpoint can be used to fetch the [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) prices of available off-chain assets in exchange for a Stellar asset and vice versa.
+
+#### Request
+
+Only one of the following parameters should be included in the request.
+
+Name | Type | Description
+-----|------|------------
+`asset` | string | (optional) The [SEP-32](sep-0032.md) address of the Stellar asset you want to get the price of available off-chain currencies in. Lumens should be specified as `XLM`.
+`currency` | string | (optional) The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) code of the currency you want to get the price of available Stellar assets in.
+
+#### Response
+
+Only one of the following attributes should be included in the response.
+
+Name | Type | Description
+-----|------|------------
+`assets` | array | (optional) An array of objects containing the available Stellar assets and their prices in terms of the `currency` provided in the request.
+`currencies` | array | (optional) An array of objects containing the available currencies and their prices in terms of the `asset` provided in the request.
+
+`assets` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`asset` | string | Same as the definition of `asset` in the request.
+`price` | string | The price of one unit of `asset` in terms the `currency` sent in the request.
+`decimals` | integer | The number of decimals needed to represent asset.
+
+`currencies` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`currency` | string | Same as the definition of `currency` in the request.
+`price` | string | The price of one unit of `currency` in terms of the `asset` sent in the request.
+`decimals` | integer | The number of decimals needed to represent the currency.
+
+### GET Price
+
+This endpoint can be used to fetch the indicative price of a Stellar asset and currency [pair](https://en.wikipedia.org/wiki/Currency_pair).
+
+#### Request
+
+Name | Type | Description
+-----|------|------------
+`base` | string | The base asset or currency. Ex. `USDC:circle.io`, `ARS`
```

Comment:
❓ What's the significance of the `ARS` example that has no issuer? Is issuer optional?

Reply:
ARS is the "external" asset as defined by [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217).

Reply:
Got it. This is not clear because there could be an ARS asset on Stellar too. What do you think about `iso4217:ARS` for referencing currencies? I'm not sold on the idea myself but it would make the API explicit when referring to assets or currencies or other things. Similar to my other comment.